### PR TITLE
Only download libonig if stack is not scalingo-14

### LINKED ISSUE #118: libonig is not available for scalingo-14
And it's not an issue because libonig is only mandatory starting with PHP 7.4 and this version of PHP is not available with scalingo-14.

### PR DESCRIPTION
**Pull request by EtienneM**
Fix #118

### CODE REVIEW

File: support/package_php [hunk 1]
```diff
@@ -41,11 +41,13 @@ if [ "$STACK" = "scalingo-14" ] ; then
   libzip_flag="--with-libzip=/app/vendor/libzip"
 fi
 
-echo "-----> Downloading dependency libonig ${libonig_version}"
-mkdir /app/vendor/libonig -p
-curl -L -o - "${SWIFT_URL}/package/libonig-${libonig_version}.tgz" | tar -C /app/vendor/libonig -xvz
-export C_INCLUDE_PATH="$C_INCLUDE_PATH:/app/vendor/libonig/include"
-export PKG_CONFIG_PATH="/app/vendor/libonig/lib/pkgconfig:$PKG_CONFIG_PATH"
+if [ "$STACK" != "scalingo-14" ] ; then
```
Comment (johnsudaar) [inline support/package_php]: Isn't `scalingo` the name of `scalingo-14` stack ?

Comment (johnsudaar) [reply support/package_php]: Should this also be ran if STACK is not fefined ?

Comment (EtienneM) [reply support/package_php]: > Isn't `scalingo` the name of `scalingo-14` stack ?

No:

```
╰─$ ascalingo -a phpmyadmin apps-info 
+----------------+-------------+
|    SETTINGS    |    VALUE    |
+----------------+-------------+
| Force HTTPS    | true        |
| Sticky Session | false       |
| Stack          | scalingo-14 |
| Status         | running     |
+----------------+-------------+
╭─emichon@biniou ~ 2.6.5
╰─$ ascalingo -a phpmyadmin run bash 
[...]

[14:07] Scalingo:phpmyadmin ~ $ echo $STACK
scalingo-14
```

> Should this also be ran if STACK is not fefined ?

Yes, I will update it

Comment (johnsudaar) [reply support/package_php]: Weird. Because on line 376 we test if the stack == "SCALINGO". Is that a bug ?

Comment (EtienneM) [reply support/package_php]: Line 376 of which file?

Comment (EtienneM) [reply support/package_php]: Moreover, from what I know, this script is only called from our local dev environment to package a new PHP version. And our script which calls this one defines the stack as being `scalingo-14`.

**Comment by johnsudaar** [comment]
> > Should this also be ran if STACK is not fefined ?
> 
> Yes, I will update it
???

**Pull request merged**